### PR TITLE
Andrew

### PR DESCRIPTION
**Pull request by ABodey**
#### Single-line Summary
**Today, Erdem and Christy and I paired together. It took about 3 hours**

#### Reflect and summarize on your process for each `TODO` item :  
  1. First, we started on the connection string
  2. Next, we worked on the route.js file
  3. Next, we worked on the article controler file
  4. Next, we worked on the about controler file
  5. Next, we worked on refactoring the html file
  6. Next, we worked on deleting the handlemain navs

#### Checklist (before submitting, fill in each set of square brackets with an 'x')
- [x] We have titled the Pull Request similar to our branch name (ex: 'brian-rick').
- [x] This PR includes commits from both myself and my partner; e.g. We followed good pair programming practices by switching driver/navigator roles.
- []x There is no extraneous, unrelated code included in this PR.
- [x] We have summarized our `TODO:` process above.

### CODE REVIEW

File: starter-code/public/index.html [hunk 1]
```diff
@@ -36,9 +36,9 @@
       <nav class="main-nav">
         <div class="icon-menu"></div>
         <ul>
-          <!-- TODO: Refactor the href attributes in the Home and About anchor tags to navigate users to our new client-side routes -->
-          <li class="tab" data-content="articles"><a href="#" class="icon-home"> Home</a></li>
-          <li class="tab" data-content="about"><a href="#" class="icon-address-book"> About</a></li>
+          <!-- DONE: Refactor the href attributes in the Home and About anchor tags to navigate users to our new client-side routes -->
```
Comment (JPGothro) [inline starter-code/public/index.html]: REQUEST: when you complete a "TODO" item in the assignment, do not change it to "DONE", but you can change it to "TODONE". I search based on "TODO", so I can see all the items that way (done or not). 
Thank you!

File: starter-code/public/index.html [hunk 2]
```diff
@@ -66,7 +66,7 @@
           Welcome to my Blog
         </header>
         <section>
-          <h1>Some of my intrests:</h1>
+          <h1>Some of my interests:</h1>
```
Comment (JPGothro) [inline starter-code/public/index.html]: Nailed the spell check!

File: starter-code/public/scripts/views/articleView.js [hunk 3]
```diff
@@ -82,14 +73,12 @@
...
-    /* TODO: Remember to also remove any invocations of handleMainNav... */
+    /* DONE: Remember to also remove any invocations of handleMainNav... */
     articleView.populateFilters();
     articleView.handleCategoryFilter();
     articleView.handleAuthorFilter();
-    articleView.handleMainNav();
     articleView.setTeasers();
   };
 
-  Article.fetchAll(articleView.initIndexPage);
```
Comment (JPGothro) [inline starter-code/public/scripts/views/articleView.js]: Nice.

File: starter-code/public/index.html (hunk 1 above)
Comment (kbrumer) [inline starter-code/public/index.html]: I think I might have told them that. I like marking them as `DONE` so I know what I have finished, but `TODONE` is good too.

**Comment by kbrumer** [comment]
🥇

**Pull request closed without merging**